api/tests/core/educational/test_models.py

```diff
@@ -695,20 +695,20 @@ def test_get_displayed_status_for_offer_with_cancelled_booking(self):
 
 
 class CollectiveOfferTemplateDisplayedStatusTest:
-    @pytest.mark.parametrize(
-        "status",
-        [
-            CollectiveOfferDisplayedStatus.ACTIVE,
-            CollectiveOfferDisplayedStatus.ARCHIVED,
-            CollectiveOfferDisplayedStatus.DRAFT,
-            CollectiveOfferDisplayedStatus.INACTIVE,
-            CollectiveOfferDisplayedStatus.REJECTED,
-            CollectiveOfferDisplayedStatus.PENDING,
-        ],
-    )
+    @pytest.mark.parametrize("status", set(COLLECTIVE_OFFER_TEMPLATE_STATUSES) - {CollectiveOfferDisplayedStatus.ENDED})
     def test_get_offer_displayed_status(self, status):
         offer = factories.create_collective_offer_template_by_status(status)
+        assert offer.displayedStatus == status
 
+    def test_get_offer_displayed_status_ended(self):
```

Comment:
Plutôt que mettre, je trouve cela plus lisible d'overrider le paramêtre (même si cela ne sert à rien en pratique)

```suggestion
    @override_features(ENABLE_COLLECTIVE_NEW_STATUSES=False)
    def test_get_offer_displayed_status_ended(self):
```

Reply:
pour moi si on lit un override c'est que la valeur change, si on suit cette logique on devrait mettre override_features avec =False sur tous les tests

Reply:
Mais c'est relativement peu important